sdk/python/feast/infra/contrib/grpc_server.py

```diff
@@ -0,0 +1,55 @@
+from concurrent import futures
```

Comment:
any thoughts on combining this with the "feature server", which can optionally be reused here? is the main reason to separate it out because you want to make a grpc server?

Reply:
i.e. https://github.com/feast-dev/feast/blob/master/sdk/python/feast/feature_server.py#L82

Reply:
Yes, we maintain gRPC connectors so HTTP is out of support.

Reply:
refactored to have similar logic with regular feature server. ✅